assets/js/src/__init.coffee

```diff
@@ -43,5 +43,6 @@ Uno.app.dataset.device = Uno.device()
 # window global properties
 $('#profile-title').text window.profile_title if window.profile_title
 $('#profile-resume').text window.profile_resume if window.profile_resume
+$('#post-list-headline').text window.post_list_headline if window.post_list_headline
```

Comment:
Do you think that `posts_headline` could be a better variable name?

Same for id selector... what do you think?